Readme.md

```diff
@@ -54,13 +54,17 @@ To obtain an API key:
   * If the API Manager page isn't already open, open the left side menu and select **API Manager**.
   * On the left, choose **Credentials**.
   * Click **Create credentials** and then select **API key**.
-  
-Copy this API key to the **APIKey** field under **SafeBrowsing** in the configuration file.
+  * Copy this API key to the **APIKey** field under **SafeBrowsing** in the configuration file.
```

Comment:
I didn't know we had to enable the api, and struggled for awhile. I've added the instructions here.